SMLHelper/Legacy/Utility.cs

```diff
@@ -22,15 +22,15 @@ public class Utility
 
         public static void AddBasicComponents(ref GameObject _object, string classId)
         {
-            var rb = _object.AddComponent<Rigidbody>();
+            Rigidbody rb = _object.AddComponent<Rigidbody>();
```

Comment:
This is another example of the type being explicit on the right-hand side.  
The type in the constructor as part of the generic definition.

This can be `var`.

Reply:
I left another comment with the general `var` guidelines.  
The idea is to minimize how verbose the code gets without sacrificing readability.